auth-service/src/main/resources/templates/thymeleaf/set-password-success.html

```diff
@@ -0,0 +1,34 @@
+<!DOCTYPE html>
+<html lang="en" class="govuk-template " xmlns:th="http://www.thymeleaf.org">
+
+<head th:replace="~{fragments/template.html :: head (title='DELTA | Set Password')}">
+    <title>DELTA | Set Password</title>
+</head>
+
+<body class="govuk-template__body govuk-body">
+<a href="#main-content" class="govuk-skip-link" data-module="govuk-skip-link">Skip to main content</a>
+<header th:replace="~{fragments/template.html :: header}"></header>
+
+<!--/*@thymesVar id="deltaUrl" type="java.lang.String" */-->
+
+<div class="govuk-width-container" id="setPasswordSuccessPage">
+    <main class="govuk-main-wrapper " id="main-content" role="main">
+        <h1 class="govuk-heading-xl">Registration completed</h1>
+        <div class="govuk-grid-row">
+            <div class="govuk-grid-column-two-thirds">
+                <div class="govuk-panel">
+                    <h1>
+                        Your password has been set and your registration is completed
+                    </h1>
+                    <a th:href="${deltaUrl}" role="button" draggable="false"
+                       class="govuk-button govuk-button--secondary">
+                        Log in to DELTA
```

Comment:
```suggestion
                    <a th:href="${deltaUrl}/login" role="button" draggable="false"
                       class="govuk-button govuk-button--secondary">
                        Sign in to DELTA
```

For consistency with the error page which has a similar button